tests/storages_tests/test_journal.py

```diff
@@ -116,3 +128,107 @@ def test_pop_waiting_trial_multiprocess_safe() -> None:
                 if trial_id is not None:
                     trial_id_set.add(trial_id)
         assert len(trial_id_set) == num_enqueued
+
+
+@pytest.mark.parametrize("log_storage_type", LOG_STORAGE_SUPPORTING_SNAPSHOT)
+def test_load_snapshot(log_storage_type: str) -> None:
+    with JournalLogStorageSupplier(log_storage_type) as storage:
+        assert isinstance(storage, optuna.storages.JournalRedisStorage)
```

Comment:
`test_load_snapshot` seems be able to test all storages that implements `BaseJournalLogSnapshot`.

```suggestion
        assert isinstance(storage, optuna.storages._journal.BaseJournalLogSnapshot)
```